### ISSUE TITLE
Header styles inconsistent across page loads

### ISSUE THREAD

**Issue by galvarez421** [PurdueMarketingAndMedia/purdueTemplates-2015#113]
Sometimes in Chrome the header styles are different across page (re)loads. More specifically, the space between the Purdue logo and the vertical bar to its right varies in size.

**Correct**:
![normal](https://user-images.githubusercontent.com/12755056/81429497-e338e780-912b-11ea-822a-ae116ef289e2.PNG)

**Incorrect**
![bug](https://user-images.githubusercontent.com/12755056/81429568-fba90200-912b-11ea-958c-502749018bef.PNG)

I am not able to reproduce this issue in Firefox, but I have reproduced it in Opera, so perhaps it's a Chromium-specific issue. The issue can be reproduced in Chrome by repeatedly reloading https://purduemarketingandmedia.github.io/purdueTemplates-2015/builds/production/html/mm/templates/office/office-tiles-4.html. Resizing the viewport seems to fix the issue. Often it will seem that the incorrect styling is used. If I throttle to Slow 3G in chrome, it will show the incorrect styling is used first but ultimately the rendering ends with the correct styling.

**Comment by jyoung295** [comment]
Yes we noticed this in chrome during development, but it only occurs during loading and snaps to the correct size after the page completes loading. We never figured out why it happens but has to be something to do with using an svg object for the mark.

That being said I haven't been able to reproduce an occurrence of the spacing staying at the larger size after the page finishes loading on a live site. See the most recent [commencement site](https://www.purdue.edu/commencement/spring2020/index.php).
